packages/gridproxy_client/src/modules/gateways.ts

```diff
@@ -46,6 +46,7 @@ export interface GPUCard {
   vendor: string;
   device: string;
   contract: number;
+  updated_at: number;
```

Comment:
why is this added?

Reply:
I replaced the cards array we add to GridNode with the gpu coming from grid proxy directly. Check https://gridproxy.dev.grid.tf/nodes/161/gpu